cmd/daemon/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,26 @@
+package kubernetes
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestMisingLabelsIsNotCorrectlyAnnotated(t *testing.T) {
```

Comment:
This is a case where table tests are the common choice in Go.
To me at least, this makes the cases tested more clear.

```go
func TestIsCorrectlyAnnotated(t *testing.T) {
	tt := []struct {
		name       string
		controlled string
		artifactID string
		author     string
		correct    bool
	}{
		{
			name:       "only controlled",
			controlled: "true",
			artifactID: "",
			author:     "",
			correct:    false,
		},
		{
			name:       "all good",
			controlled: "true",
			artifactID: "1",
			author:     "platon",
			correct:    true,
		},
	}
	for _, tc := range tt {
		t.Run(tc.name, func(t *testing.T) {
			correct := isCorrectlyAnnotated(map[string]string{
				"lunarway.com/controlled-by-release-manager": tc.controlled,
				"lunarway.com/artifactId":                    tc.artifactID,
				"lunarway.com/author":                        tc.author,
			})

			assert.Equal(t, tc.correct, correct)
		})
	}
}
```